src/js/vre/chapter31/config/form.js

```diff
@@ -76,9 +89,40 @@ const formConfig = {
         workInformation: {
           path: 'work-information',
           title: 'Work Information',
+          uiSchema: {
+            'view:isWorking': {
+              'ui:title': 'Are you working?',
+              'ui:widget': 'yesNo'
+            },
+            employer: {
+              'ui:title': 'Employer name',
+              'ui:required': (formData) => formData['view:isWorking'],
+              ...expandIfWorking
+            },
+            jobDuties: {
+              'ui:title': 'Job duties',
+              ...expandIfWorking
+            },
+            monthlyIncome: _.merge(
```

Comment:
Dunno why I used spread and `merge` here. Can update if there is a preference.

Reply:
I think being consistent is good, but either approach seems fine to me.